tests/system_tests_qdmanage.py

```diff
@@ -208,10 +208,6 @@ def test_get_types(self):
         out = json.loads(self.run_qdmanage("get-types"))
         self.assertEqual(len(out), TOTAL_ENTITIES)
 
-    def test_get_attributes(self):
-        out = json.loads(self.run_qdmanage("get-attributes"))
-        self.assertEqual(len(out), 28)
-
```

Comment:
Test of the same name is defined right below. Since the latter shadows the former, I am deleting the former. Probably a copypaste error anyways.